### PR TITLE
feat[added video details page]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
video-details

### CODE REVIEW

File: src/Components/VideoDetailPage/VideoDetailPage.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { useParams } from "react-router-dom";
+import ReactPlayer from "react-player";
+import axios from "axios";
+
+import { VideoDetailsSection } from "./VideoDetailsSection";
+import "./video-detail-page.css";
+
+export const VideoDetailPage = () => {
+  const [videoDetails, setVideoDetails] = useState(null);
+  //const playerRef = useRef(null);
```
Comment (anuj-kumary) [inline src/Components/VideoDetailPage/VideoDetailPage.js]: Remove unwanted comments

File: src/Components/VideoDetailPage/VideoDetailsSection.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              <i class="fas fa-folder-plus"></i>
+            </button>
+            <button className="btn large-btn video-action-button">
+              <i className="far fa-clock text-large"></i>
+            </button>
+            <button className="btn large-btn video-action-button">
+              <i class="far fa-edit"></i>
+            </button>
+            <button className="btn large-btn video-action-button">
+              <i class="fas fa-share"></i>
```
Comment (anuj-kumary) [inline src/Components/VideoDetailPage/VideoDetailsSection.js]: Here change class to className

**Review by anuj-kumary — APPROVED** [review]
Code is very clean and also good use of naming conventions.

**Review by vermasavita — APPROVED** [review]
Good work.

**Pull request merged**